api/tests/unit/segments/test_unit_segments_views.py

```diff
@@ -1556,3 +1557,51 @@ def test_include_feature_specific_query_filter__false(
     # Then
     assert response.json()["count"] == 1
     assert [res["id"] for res in response.json()["results"]] == [segment.id]
+
+
+def test_clone_segment(
+    project: Project,
+    admin_client: APIClient,
+    segment: Segment,
+) -> None:
+    # Given
+    url = reverse(
+        "api-v1:projects:project-segments-clone", args=[project.id, segment.id]
+    )
+    new_segment_name = "cloned_segment"
+    data = {
+        "name": new_segment_name,
+    }
+    # When
+    response = admin_client.post(
+        url, data=json.dumps(data), content_type="application/json"
+    )
```

Comment:
If you use an actual instance of `APIClient` rather than `admin_client`, the JSON bits here should be built-in.

Reply:
I am sorry, the above was me assuming `admin_client` was [pytest-django's](https://pytest-django.readthedocs.io/en/latest/helpers.html#admin-client-django-test-client-logged-in-as-admin) `admin_client`. Rather, it is an instance of DRF's `APIClient`, as per this fixture:

https://github.com/Flagsmith/flagsmith/blob/fd1e63cab0e2ed50bdf951511c7ce3248297820f/api/conftest.py#L234

Please ignore the above. :)